src/catalogue/items/catalogueItemsDialog.component.tsx

```diff
@@ -605,6 +605,39 @@ function CatalogueItemsDialog(props: CatalogueItemsDialogProps) {
     [errorMessages, propertyErrors]
   );
 
+  const options = (): Array<Manufacturer & { isRecent: string }> => {
+    const classifiedManufacturers = manufacturerList
+      ? manufacturerList.map((option) => {
+          return {
+            ...option,
+            isRecent: 'A-Z',
+          };
+        })
+      : [];
+
+    //duplicating the recent manufacturers, so that they appear twice.
+    const date_now = new Date().getTime();
+    const recentManufacturers = classifiedManufacturers
+      .filter((option) => {
+        const created_date = new Date(option.created_time).getTime();
+        const TEN_MINUTES = 1000 * 60 * 10;
```

Comment:
Can you move `const TEN_MINUTES = 1000 * 60 * 10;` to the top of the file below the imports please

Reply:
I think it would also be nice if instead of being `TEN_MINUTES` it was named something more specific to what its for e.g. `RECENT_MANUFACTURER_MAX_AGE_MINUTES`?